content/docs/command-reference/plots/index.md

```diff
@@ -280,7 +304,157 @@ file:///Users/usr/src/dvc_plots/index.html
 ![](/img/plots_show_confusion.svg)
 
 > A confusion matrix [template](/doc/command-reference/plots#plot-templates) is
-> predefined in DVC (found in `.dvc/plots/confusion.json`).
+> predefined in DVC.
+
+We can use `confusion_normalized` template to normalize the results:
+
+```dvc
+$ dvc plots show classes.csv --template confusion_normalized -x actual -y predicted
+file:///Users/usr/src/dvc_plots/index.html
+```
+
+![](/img/plots_show_confusion_normalized.svg)
+
+## Example: simple plot definition
```

Comment:
My feeling that we should create a large table with three-four columns - data, template, spec (dvc.yaml), image - and put a lot of examples there - from some basic regular things, to more advanced. So that people can quickly start, search for it, copy paste, etc

Reply:
Some more organization would definitely be useful, and more examples would be great.

For now, could we at least try to break it up so it's not one huge list of examples? Maybe have different subsections. It seems like there's a bunch of path-based examples, followed by dvc.yaml examples, and then finally a custom html template example.